Documentation/Ctrl/Properties/PreviewRenderer.rst

```diff
@@ -0,0 +1,58 @@
+.. include:: /Includes.rst.txt
+.. _ctrl-reference-previewrenderer:
+
+===============
+previewRenderer
+===============
+
+.. confval:: previewRenderer
+
+   :Path: $GLOBALS['TCA'][$table]['ctrl']
+   :type: string
+   :Scope: Display
+
+   Configure a backend preview for a content element.
+
+Examples
+========
+
+Have also a look at :ref:`t3coreapi:ConfigureCE-Preview` for more details.
+
+Use for any record in a table
+-----------------------------
+
+This specifies the preview renderer to be used for any record in :sql:`my_table`:
```

Comment:
should we use an allowed table name format such as tx_myextensions_table?

Reply:
Done